vtex-apps/return-app
# feat: add pickup points as as a delivery option

What this PR changes:
Backend:
Functionality to fetch near pickup points by geoCoordinates. The api Im reaching is this one https://developers.vtex.com/vtex-rest-api/reference/listpickupppointsbylocation

Frontend and admin:
Functionality to add near pickup points as a delivery option to the user. The default behaviour is to not show this options, in order to display them we have a new settings on the return settings app (admin).


How to test it:
[this workspace](https://pickuppoints--vtexspain.myvtex.com/account#/my-returns).
On this workspace create a return request and click on the toggle aside the default delivery option. Select a pickup point, go to the next step and the preview you can see that the address is updated with the pickup point data.
